src/parseFunctions.js

```diff
@@ -0,0 +1,23 @@
+import {keyTypes} from './constants'
```

Comment:
I had to change to ./constanst.js in the other case I get this error:

```
node:internal/errors:477
    ErrorCaptureStackTrace(err);
    ^

Error [ERR_MODULE_NOT_FOUND]: Cannot find module '/home/spique/Documents/MontevideoTech/cmcd-validator/src/constants' imported from /home/spique/Documents/MontevideoTech/cmcd-validator/src/parseFunctions.js
    at new NodeError (node:internal/errors:387:5)
    at finalizeResolution (node:internal/modules/esm/resolve:404:11)
    at moduleResolve (node:internal/modules/esm/resolve:965:10)
    at defaultResolve (node:internal/modules/esm/resolve:1173:11)
    at nextResolve (node:internal/modules/esm/loader:173:28)
    at ESMLoader.resolve (node:internal/modules/esm/loader:852:30)
    at ESMLoader.getModuleJob (node:internal/modules/esm/loader:439:18)
    at ModuleWrap.<anonymous> (node:internal/modules/esm/module_job:76:40)
    at link (node:internal/modules/esm/module_job:75:36) {
  code: 'ERR_MODULE_NOT_FOUND'
}
```
I'm running on Node.js v18.6.0